source/develop/developer-guide/engine/engine-development-environment.html.md

```diff
@@ -20,131 +20,127 @@ The purpose of this page is primarily to align the community experience with the
 
 ### Prerequisites
 
-#### RPM-Based Systems
+### RPM-Based Systems
 
-##### Set Up the Snapshot Repository
+1. Set Up the Snapshot Repository
 
-###### Automatically
+    * Automatically
 
-Run the following command:
+        Run the following command:
 
-`# yum install `[`http://resources.ovirt.org/pub/yum-repo/ovirt-releaseXY.rpm`](http://resources.ovirt.org/pub/yum-repo/ovirt-releaseXY.rpm)
+                # yum install http://resources.ovirt.org/pub/yum-repo/ovirt-releaseXY.rpm
 
-<b>NOTE:</b> This does not work for master because it is available after release.
+        <b>NOTE:</b> This does not work for master because it is available after release.
 
-###### Manually
+    * Manually
 
-Create the file `/etc/yum.repos.d/ovirt-snapshots.repo`, and replace `@distro@` in the following code block with `fc` for Fedora or `el` for RHEL or an equivalent distribution.
+        Create the file `/etc/yum.repos.d/ovirt-snapshots.repo`, and replace `@distro@` in the following code block with `fc` for Fedora or `el` for RHEL or an equivalent distribution.
 
-      [ovirt-snapshots]
-      name=local
-      baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot/rpm/@distro@$releasever
-      enabled=1
-      gpgcheck=0
-      priority=10
+          [ovirt-snapshots]
+          name=local
+          baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot/rpm/@distro@$releasever
+          enabled=1
+          gpgcheck=0
+          priority=10
 
-      [ovirt-snapshots-static]
-      name=local
-      baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/@distro@$releasever
-      enabled=1
-      gpgcheck=0
-      priority=10
+          [ovirt-snapshots-static]
+          name=local
+          baseurl=http://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/@distro@$releasever
+          enabled=1
+          gpgcheck=0
+          priority=10
 
-##### Set Up the PatternFly Repository
```

Comment:
Correct, patternfly is delivered via ovirt-js-dependencies. It's one of 5 or 6 libraries now, and we're just packaging them all up in there.